src/parse.c

```diff
@@ -4840,6 +4840,10 @@ static tree_t p_interface_function_specification(void)
    // [ pure | impure ] function designator
    //    [ [ parameter ] ( formal_parameter_list ) ] return type_mark
 
+   // 2019:
+   // [ pure | impure ] function designator
+   //    [ [ parameter ] ( formal_parameter_list ) ] return [ return_identifier of ] type_mark
```

Comment:
Can you wrap the lines at 80 characters? (Here and below.)

```suggestion
   // [ pure | impure ] function designator
   //    [ [ parameter ] ( formal_parameter_list ) ] 
   //    return [ return_identifier of ] type_mark
```